fastlane/lib/fastlane/server/command.rb

```diff
@@ -0,0 +1,72 @@
+require 'json'
+
+module Fastlane
+  class Argument
+    def initialize(json: nil)
+      @name = json['name']
+      @value = json['value']
+      @value_type = json['value_type']
+    end
+
+    def is_named
+      return @name.to_s.length > 0
+    end
+
+    def inspect
+      if is_named
+        return "named argument: #{name}, value: #{value}, type: #{value_type}"
+      else
+        return "unnamed argument value: #{value}, type: #{value_type}"
+      end
+    end
+
+    attr_reader :name
+    attr_reader :value
+    attr_reader :value_type
+  end
+
+  class Command
+    def initialize(json: nil)
+      command_json = JSON.parse(json)
+      @method_name = command_json['methodName']
+      @class_name = command_json['className']
+      @command_id = command_json['commandID']
+
+      args_json = command_json['args'] ||= []
+      @args = args_json.map do |arg|
+        Argument.new(json: arg)
+      end
+    end
+
+    def target_class
+      unless class_name
+        return nil
+      end
+
+      # rubocop:disable Security/Eval
+      return eval(class_name)
```

Comment:
Loading a class with `eval` seems a little off. Couldn't we use the following 
```ruby
Fastlane::Actions.const_get(class_name)
```
^ in this case we'd load a class that's inside the module `Fastlane::Actions`